pkg/pub_integration/test/fake_sign_in_test.dart

```diff
@@ -46,8 +46,8 @@ void main() {
       final browserSession = await fakeTestScenario.createAnonymousTestUser();
       String? firstSessionId;
       // sign-in page
-      await browserSession.withBrowserPage(
-        (page) async {
+      await browserSession.withBrowserPage((page) async {
+        {
           final rs = await page.gotoOrigin('/sign-in?fake-email=user@pub.dev');
```

Comment:
Isn't the idea that `withBrowserPage` will return a browser instance that is already signed-in for the given user?